opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -388,8 +402,12 @@ impl BatchLogProcessor {
     where
         E: LogExporter + Send + Sync + 'static,
     {
-        let (message_sender, message_receiver) = mpsc::sync_channel(config.max_queue_size);
+        let (logs_sender, logs_receiver) = mpsc::sync_channel::<LogsData>(config.max_queue_size);
+        let (message_sender, message_receiver) = mpsc::sync_channel::<BatchMessage>(64); // Is this a reasonable bound?
```

Comment:
64 for signal (flush, shutdown, and trigger export). I don't think user should be doing flush/shutdown too often. But could there be scenario if the code-generated signal to trigger export can ever get flooded.

Reply:
> But could there be scenario for the code-generated signal to trigger export can ever get flooded.

There can never be more than one code-generated signal in the channel. Threads calling `emit()` cannot send the `ExportLog` signal if there's one already in the channel.

Reply:
Make sense. Probably add a TODO to decide on the optimal value.